internal/util/slices.go

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package util
+
+// Map maps a slice of one type into a slice of another using the provided map
+// function
+func Map[T, U any](in []T, f func(T) U) []U {
```

Comment:
Is there a need to move this into a separate package? Are we planning on using this elsewhere? If not I would rather leave this in the test package. If we do expect to use this in other places, lets put this in a more descriptive package name and avoid using `util`.

Reply:
I was using it previously but not anymore. Dropped this change's commit -- this func is now back in its original place in the test package